src/middlewared/middlewared/plugins/network.py

```diff
@@ -1076,24 +1072,29 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
                     'Failover needs to be disabled to perform network configuration changes.'
                 )
 
-            found = True
-            for i in (
-                'failover_critical', 'failover_group', 'failover_aliases', 'failover_vhid',
-            ):
-                if i not in data:
-                    verrors.add(
-                        f'{schema_name}.{i}',
-                        'This attribute is required when configuring HA.',
-                    )
-                    found = False
-            if found:
-                if len(data['aliases']) != len(data['failover_aliases']):
-                    verrors.add(
-                        f'{schema_name}.failover_aliases',
-                        'Number of IPs must be the same between controllers.',
-                    )
```

Comment:
Just saw this now.. did we lose this check?

Reply:
Doesnt have to compare controllers, just the number of ips for node A vs node B

Lets add it back.

Reply:
As well in 11.3 if we ended up removing it